templates/designToConfigParser.jinja

```diff
@@ -103,10 +103,34 @@ bool {{functionPrefix}}configure{{class_name}} (
   mapping addressActiveControl,
 	mapping connectionSettings)
 {
-  DebugTN("Configure.{{class_name}} called");
+  DebugTN("Configure.{{class_name}} called from XML");
```

Comment:
This can be kept as is as anyway configureFromName is to be called thereafter and will produce a contradictory message i.e. called without XML